sdk/communication/Azure.Communication.CallAutomation/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Release History
 
+## 1.0.1-beta.1 (Unreleased)
```

Comment:
Beta.2

Reply:
Forgot to remove this - have undone the change entirely